spyder/app/mainwindow.py

```diff
@@ -274,10 +274,7 @@ def qt_message_handler(msg_type, msg_log_context, msg_string):
 # =============================================================================
 # Dependencies
 # =============================================================================
-QDARKSTYLE_REQVER = '>=2.6.4'
-dependencies.add("qdarkstyle", "qdarkstyle",
-                 _("Dark style for the entire interface"),
-                 required_version=QDARKSTYLE_REQVER)
+dependencies.declare_dependencies()

```

Comment:
Let's move this to the method called `report_missing_dependencies`, present also here in the main window, like this:

```python
def report_missing_dependencies(self):
    """Show a QMessageBox with a list of missing hard dependencies"""
    dependencies.declare_dependencies()
    missing_deps = dependencies.missing_dependencies()
    ...
```